Kitsune/ClubPenguin/Handlers/Game/Four.php

```diff
@@ -0,0 +1,227 @@
+<?php
+namespace Kitsune\ClubPenguin\Handlers\Game;
+
+use Kitsune\Logging\Logger;
+use Kitsune\ClubPenguin\Packets\Packet;
+
+use Kitsune\ClubPenguin\Handlers\Game\FindFour;
+
+trait Four
+{
```

Comment:
Make your brackets consistent with the rest of the codebase. Keep them on the same line as whatever language construct you're declaring (classes, functions, etc).